iocore/cache/Cache.cc

```diff
@@ -2070,6 +2070,16 @@ AIO_Callback_handler::handle_disk_failure(int /* event ATS_UNUSED */, void *data
   for (; disk_no < gndisks; disk_no++) {
     CacheDisk *d = gdisks[disk_no];
 
+    /* find the Vol instance needed for updating per-volume stats by CACHE_INCREMENT_DYN_STAT */
+    Vol *vol;
```

Comment:
If there are multiple volumes for the same physical disk / file descriptor, does this just find the first one?